### ISSUE TITLE
How to use `seed`

### ISSUE THREAD

**Issue by spring-haru** [ucbds-infra/otter-grader#346]
**Describe your inquiry** 

I tried to figure out how to use `numpy`'s `seed` into an assignment, but got confused in Assign Format 1. I wonder if an example `.ipynb` file can be made available, illustrating how to use `default_rng`, e.g.

```
from numpy.random import default_rng
rng = default_rng(123)
x = rng.normal(size=10)
```

or revise `demo.ipynb` used in Tutorial of the manual.

An example of a command line would be helpful as well, because `otter assign` does not seem to recognize `--seed`.

**Comment by chrispyles** [comment]
You can find an example of this in Question 1.5 of [this assignment](https://github.com/ucbds-infra/otter-grader/blob/master/examples/data-88e-proj03/proj03.ipynb). Otter Assign's seeding doesn't support `numpy.default_rng`; instead, you need to see numpy and then use `np.random.*`.

```python
np.random.seed(42) # SEED
x = np.random.normal(size=10)
```

Then put the seed value (which should be the same for every `SEED` line) in your assignment config:

```yaml
generate:
    seed: 42
```

**Comment by spring-haru** [comment]
It is working now. Your advice was extremely helpful, thanks, (^o^)/

But, one thing which is not clear to me is the following. I thought that `generate:` is equivalent to `generate:false`, but  a problem seems to arise if the latter is used.

Two suggestions.
1. It would be useful for new users, like me, if a link for an example like the one you gave can be found in the document.
2. I suppose that many would like to teach code recommended rather than a legacy version. So, it would be great if `numpy.default_rng()` is supported.

I really appreciate your work and effort.

**Comment by chrispyles** [comment]
OK so I've opened #350 which implements a new way of seeding for v1 formatted notebooks. It will be included in v3.0.5.

You can find an example of how this seeding will work when it's released in Question 6 of this notebook: https://github.com/chrispyles/otter-grader/blob/exec-refactor/docs/_static/notebooks/assign-full-example-v1.ipynb

**Comment by chrispyles** [comment]
I've just release v3.0.5. You can find more in-depth documentation about this seed specification here: https://otter-grader.readthedocs.io/en/latest/otter_assign/v1/notebook_format.html#intercell-seeding
